scripts/make_gitrev.sh

```diff
@@ -35,10 +35,13 @@ EOF
 }
 trap cleanup EXIT
 
-# Check if git is installed and we are in a git repo.
-git rev-parse --git-dir > /dev/null 2>&1
+# Compute expected git dir location, as using `git` directly would traverse the
+# whole hierarchy, potentially reading from some parent git repo where theses
```

Comment:
nit: theses -> these
nit: some parent -> a parent

Reply:
Should have better follow these english courses..

Reply:
No problem!